components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -37,3 +42,8 @@ export const getRampNetworkPrefix = (chainId: string) => {
 export const formatAsDouble = (value: string): string =>
   // Removes all characters except numbers, commas and decimals
   value.replace(/[^0-9.,]+/g, '')
+
+export const isValidateUrl = (url: string) => {
+  const re = /^\s*https?:\/\//
```

Comment:
why allow `\s` here? wouldn't the URL fail to load if that were the case?

i think this would be cleaner as just `return url.startsWith('http://') || url.startsWith('https://')`

Reply:
nvm i found chrome.tabs does support the leading whitespace